components/dialog/dialog.js

```diff
@@ -332,7 +346,15 @@ export class Dialog extends HTMLElement {
     async show(header, main, footer, options) {
         const struct = {header, main, footer, options};
         this.#stack.push(struct);
-        await this.#showStruct(struct);
+        this.style.opacity = 0;
+
+        if (this.#stack.length > 1) {
```

Comment:
I don't see why we have this if-else. You call the same function in both, but on the if you wrap the #showStruct function call in a setTimeout? Please have a look at this

Reply:
removed the if else and made the adjustment necessary for the test to run without time delay.